### ISSUE TITLE
Cannot show documentIds in Content Manager

### ISSUE THREAD

**Issue by Matb85** [strapi/strapi#22053]
### Required System information

- Node.js version: v22.9.0
- NPM version: v10.8.3
- Strapi version: v5.2.0
- Database: postgres
- Operating system: Macos 15.0.1
- Is your project Javascript or Typescript: Typescript

### Describe the bug

Cannot display documentId in the content manager 
### Steps to reproduce the behavior

1. Go to Content Manager
2. View a collection
3. Try to show the documentId field

### Expected behavior

I should be able to see the "documentId" field of the entry. 

AFAIK, each language version of an entry has a different "documentId" then please provide a way to view them all.

Alternatively, allow to query documents by the old "id" field so that knowing "documentId" is not necessary

### Screenshots

See? no "documentId" option

![Screenshot 2024-11-02 at 22 47 08](https://github.com/user-attachments/assets/5e53b8dd-6f2f-4666-9890-e745ad8ebc54)

### Additional context

Currently, it is very difficult to work with the api, it is harder to check the documentIds but at the same time I caanot use the old "id" field to query an entry

**Comment by surfjedi** [comment]
Yeah its redoculous we can't see documentId in the manager. I second this

**Comment by verancegold** [comment]
Agreed. Would make my work a lot more streamlined!

**Comment by TJBachorz** [comment]
This is making life difficult, a fix would be greatly appreciated 🙏

**Comment by allison-hamelin** [comment]
I am seeing this issue as well.

**Comment by Matb85** [comment]
Hi, any updates on this?

**Comment by derrickmehaffy** [comment]
Related to TID 11591

**Comment by justnils** [comment]
push! why don't they show the document id?!
